### DISCUSSION TITLE
[BUG]  Discord PTT

### DISCUSSION THREAD

**Discussion by jbeach4646** [mylinuxforwork/dotfiles#294] [General]
**Describe the bug**
When install base hyprland using the arch install media, i can get Discord push to talk to work by launching it with: "env XDG_SESSION_TYPE=x11 discord"

After installing your dotfilles, something in the installation changes and this command does not work anymore. 

**To Reproduce**
1. env XDG_SESSION_TYPE=x11 discord
2. discord settings 
3. Voice & Video
4. Push to Talk
5. set keybinding(not allowing input)

**Expected behavior**
When setting keybinding, allows input.

**Distribution (please complete the following information):**
 - Arch Linux
 - Hyprland
 
**Additional context**
I dont know how to fix this, but im guessing it has something to do with Xwayland not running correctly

**Comment by mylinuxforwork** [comment]
@jbeach4646 The ML4W Dotfiles comes with own environment variables. But you can add your own in ~/.config/hypr/conf/custom.conf.

e.g. env = XDG_SESSION_TYPE, x11 discord

Can you try it?

**Comment by jbeach4646** [comment]
sadly no that doesn't work for me either. Is there a way that you run discord with PTT? If there is a better workaround i would be all for it too, i just used this one in the past and it worked.

**Comment by mylinuxforwork** [comment]
@jbeach4646 Thanks for testing? Where have you set this environment variable before? Maybe you can create a script with setting this and execute it on startup?

**Comment by jbeach4646** [comment]
In the past i added it directly into /usr/share/applications/disord.desktop. Probably the wrong way to do it in linux, but im still new and somehow it works.

Exec=env XDG_SESSION_TYPE=x11 /usr/bin

**Comment by mylinuxforwork** [comment]
@jbeach4646 I would convert this issue to a discussion because it's not a ML4W Dotfiles issue.
